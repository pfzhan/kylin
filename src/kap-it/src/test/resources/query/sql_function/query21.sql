-- https://github.com/Kyligence/KAP/issues/13613

select substr(initcapb(lstg_format_name),2) a
from test_kylin_fact