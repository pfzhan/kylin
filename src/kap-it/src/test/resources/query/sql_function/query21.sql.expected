-- https://github.com/Kyligence/KAP/issues/13613

select substr(initcap(lstg_format_name),2) a
from test_kylin_fact